platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -417,13 +417,22 @@ public Response processLogin(@QueryParam(SAML_REQ) String samlRequest,
                 SamlProtocol.createStatus(statusCode), authnRequest.getID(), samlToken);
     }
 
-    private synchronized Element exchangeCookieForAssertion(HttpServletRequest request) {
+    private synchronized  Element exchangeCookieForAssertion(HttpServletRequest request) {
```

Comment:
No big deal, but it looks like you have 2 spaces between 'synchronized' and 'Element'... Did you not reformat the code after changes?